m2cgen/interpreters/interpreter.py

```diff
@@ -100,27 +101,8 @@ def __init__(self, cg, feature_array_name="input"):
         self.with_vectors = False
         self.with_math_module = False
 
-    def interpret_if_expr(self, expr, if_var_name=None, **kwargs):
-        if if_var_name is not None:
-            var_name = if_var_name
-        else:
-            var_name = self._cg.add_var_declaration(expr.output_size)
-
-        def handle_nested_expr(nested):
-            if isinstance(nested, ast.IfExpr):
-                self._do_interpret(nested, if_var_name=var_name, **kwargs)
-            else:
-                nested_result = self._do_interpret(nested, **kwargs)
-                self._cg.add_var_assignment(var_name, nested_result,
-                                            nested.output_size)
-
-        self._cg.add_if_statement(self._do_interpret(expr.test, **kwargs))
-        handle_nested_expr(expr.body)
-        self._cg.add_else_statement()
-        handle_nested_expr(expr.orelse)
-        self._cg.add_block_termination()
-
-        return var_name
+    def interpret_if_expr(self, expr, **kwargs):
```

Comment:
Sure, I can! But I thought it will help future contributors indicate what they should implement by reading the code only of parent class. But I'll remove two these lines without any regrets if you feel that they should not be here 🙂

Reply:
I have no strong opinion here TBH. It just seemed a bit redundant. Leaving this up to you.